release notes/v0.52.0.md

```diff
@@ -0,0 +1,148 @@
+
+This release of k6 introduces experimental support for TypeScript and ES6+ using esbuild, thanks to a new
+[compatibility mode](https://grafana.com/docs/k6/latest/using-k6/javascript-compatibility-mode/) named `experimental_enhanced`.
+
+```sh
+k6 run --compatibility-mode=experimental_enhanced script.js
+```
+
+With this new compatibility mode, the test source code is transformed using esbuild instead of Babel, which also means
+that source files with the extension _".ts"_ are loaded by esbuild's TypeScript loader,  which results in partial 
+TypeScript support: it removes the type information, but does not provide type safety.
+
+### `k6/browser` has graduated from an experimental module [#3793](https://github.com/grafana/k6/pull/3793)
+
+The browser module is now available as `k6/browser` instead of `k6/experimental/browser`. The previous `k6/experimental/browser` module will be removed on September 23rd, 2024. Please see [the migration guide](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) for more information on how to update your scripts.
+
+### `k6/browser` has now a fully Async API [browser#428](https://github.com/grafana/xk6-browser/issues/428)
+
+This release introduces a fully Async API for the `k6/browser` module. This means that nearly all the methods in the module now return promises. This change is part of the ongoing effort to make the browser module more user-friendly and easier to use. Please see [the browser documentation](https://grafana.com/docs/k6/latest/javascript-api/k6-browser) for more information on how to use the new Async API.
+
+Related Changes:
+- [browser#1310](https://github.com/grafana/xk6-browser/pull/1310), [browser#1311](https://github.com/grafana/xk6-browser/pull/1311), [browser#1312](https://github.com/grafana/xk6-browser/pull/1312), [browser#1316](https://github.com/grafana/xk6-browser/pull/1316) Migrate `Browser` APIs to async.
+- [browser#1328](https://github.com/grafana/xk6-browser/pull/1328) Migrate `BrowserContext` APIs to async.
+- [browser#1337](https://github.com/grafana/xk6-browser/pull/1337) Migrate `ElementHandle` APIs to async.
+- [browser#1367](https://github.com/grafana/xk6-browser/pull/1367) Migrate `Frame` APIs to async.
+- [browser#1366](https://github.com/grafana/xk6-browser/pull/1366) Migrate `JSHandle` APIs to async.
+- [browser#1314](https://github.com/grafana/xk6-browser/pull/1314) Migrate `Keyboard` APIs to async.
+- [browser#1332](https://github.com/grafana/xk6-browser/pull/1332) Migrate `Locator` APIs to async.
+- [browser#1323](https://github.com/grafana/xk6-browser/pull/1323) Migrate `Mouse` APIs to async.
+- [browser#1355](https://github.com/grafana/xk6-browser/pull/1355) Migrate `Page` APIs to async.
+- [browser#1348](https://github.com/grafana/xk6-browser/pull/1348) Migrate `Request` APIs to async.
+- [browser#1364](https://github.com/grafana/xk6-browser/pull/1364) Migrate `Response` APIs to async.
```

Comment:
Should we group them? Or, the current list is fine?

```suggestion
- [browser#1310](https://github.com/grafana/xk6-browser/pull/1310), [browser#1311](https://github.com/grafana/xk6-browser/pull/1311), [browser#1312](https://github.com/grafana/xk6-browser/pull/1312), [browser#1316](https://github.com/grafana/xk6-browser/pull/1316), [browser#1328](https://github.com/grafana/xk6-browser/pull/1328), [browser#1337](https://github.com/grafana/xk6-browser/pull/1337), [browser#1367](https://github.com/grafana/xk6-browser/pull/1367), [browser#1366](https://github.com/grafana/xk6-browser/pull/1366), [browser#1314](https://github.com/grafana/xk6-browser/pull/1314), [browser#1332](https://github.com/grafana/xk6-browser/pull/1332), [browser#1323](https://github.com/grafana/xk6-browser/pull/1323), [browser#1355](https://github.com/grafana/xk6-browser/pull/1355), [browser#1348](https://github.com/grafana/xk6-browser/pull/1348), [browser#1364](https://github.com/grafana/xk6-browser/pull/1364) Migrates `Browser`, `BrowserContext`, `ElementHandle`, `Frame`, `JSHandle`, `Keyboard`, `Locator`, `Mouse`, `Page`, `Request`, `Response` APIs to async.
```

Reply:
Yeah, I think that grouped it looks nicer.